lib/Controller.php

```diff
@@ -127,6 +129,7 @@ public function __construct()
         }
 
         // load config from ini file, initialize required classes
```

Comment:
Comment may need update (generally best to refactor away such comments which can be expressed as code, but for now fine, IMHO.)

```suggestion
        // load config from ini file by default, initialize required classes
```